src/rules/style.ts

```diff
@@ -36,7 +38,7 @@ module.exports = {
   // Disallows parameter properties in class constructors.
   'no-parameter-properties': false,
   // Enforces consistent object literal property quote style.
-  'object-literal-key-quotes': [true, 'as-needed'],
+  'object-literal-key-quotes': [true, 'always'],
```

Comment:
I just realized, simply overriding the rule in `tslint.json` won't be sufficient, since type checking rules won't be enforced in the npm lint script.

I've addressed that issue by creating a `./config` directory with a `full.json` and `rules.js` files similar to what we have in our other [project](http://www.nsd-fusion.com/template/images/icons/neutron.png).